index.html

```diff
@@ -0,0 +1,188 @@
+                    <li><a href="https://medium.com/@regiss05" target=”_blank”><img src="./Images/Medium.svg"
+                                alt="Medium" /></a></li>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <div class="container">
+        <div class="container-block">
+            <h1>My Recent Works</h1>
+            <section>
+                <img src="./Images/Placeholder.svg" alt="Placeholder">
+                <h2>Multi-Post Stories</h2>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                    text.
+                </p>
+                <div class="box-bot">
+                    <div>CSS</div>
+                    <div>html</div>
+                    <div>bootstrap</div>
+                    <div>Ruby</div>
+                </div>
+                <button type="button">See Project</button>
+            </section>
+            <div>
+                <form class="container-poppup">
+                    <div class="pupopp">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                            the industry's standard</p>
+                        <div class="luang">
+                            <div>html</div>
+                            <div>bootstrap</div>
+                            <div>Ruby</div>
+                        </div>
+                    </div>
+                    <button type="button">See Project</button>
+                </form>
+                <form class="container-poppup">
+                    <div class="pupopp">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                            the industry's standard</p>
+                        <div class="luang">
+                            <div>html</div>
+                            <div>bootstrap</div>
+                            <div>Ruby</div>
+                        </div>
+                    </div>
+                    <button type="button">See Project</button>
+                </form>
+                <form class="container-poppup">
+                    <div class="pupopp">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+                            the industry's standard</p>
+                        <div class="luang">
+                            <div>html</div>
+                            <div>bootstrap</div>
+                            <div>Ruby</div>
```

Comment:
- [ ] Please, use `ul` tag instead of `div` for the list of technologies used and each of them a `li` tag instead of `div`.
- Example:
```
<ul>
   <li></li>
   <li></li>
   <li></li>
</ul>